lib/detect-testing-library-utils.ts

```diff
@@ -93,17 +105,75 @@ export function detectTestingLibraryUtils<
       },
 
       /**
-       * Gets if filename being analyzed is valid or not.
-       *
-       * This is based on "testing-library/filename-pattern" setting.
+       * Determines whether filename is valid or not for current file
+       * being analyzed based on "testing-library/filename-pattern" setting.
        */
       getIsValidFilename() {
         const fileName = context.getFilename();
         return !!fileName.match(filenamePattern);
       },
 
       /**
-       * Wraps all conditions that must be met to report rules.
+       * Determines whether a given node is `getBy*` or `getAllBy*` query variant or not.
+       */
+      isGetByQuery(node) {
+        return !!node.name.match(/^get(All)?By.+$/);
```

Comment:
Aggressive query reporting enabled! It doesn't matter if it's a Testing Library built-in query or a custom one: if it matches `getBy*` query pattern, it's considered like so. Same for `queryBy*` queries, and `findBy*` in the future.